### PR TITLE
entity: multiple choice

### PR DESCRIPTION
**Pull request by arekkas**

### CODE REVIEW

**Comment by arekkas** [comment]
please :eyes: @inyono @Knorrke

**Comment by inyono** [comment]
LGTM

**Comment by arekkas** [comment]
updated logic to represent spec. kinda messy tho

File: src/assets/source/scripts/modules/serlo_multiple_choice.js [hunk 1]
```diff
@@ -5,14 +5,49 @@ define(['jquery'], function ($) {
...
+
+                $feedbackFailure.hide();
+                $feedbackSuccess.hide();
+                if (mistakes === 0) {
+                    if (feedback > 0) {
+                        $feedbackSuccess.html('<p>' + feedback.join('</p><p>') + '</p>');
+                    }
+                    $feedbackSuccess.show();
+                } else {
+                    $feedbackFailure.show();
```
Comment (inyono) [inline src/assets/source/scripts/modules/serlo_multiple_choice.js]: so there could be multiple positive feedbacks (that will all be shown) but only one single negative feedback?

Comment (arekkas) [inline src/assets/source/scripts/modules/serlo_multiple_choice.js]: yup, discussed this spec with @SimonKoehl today

Comment (arekkas) [inline src/assets/source/scripts/modules/serlo_multiple_choice.js]: usually only one positive feedback thingy should be added. if more exists, they will be shown in a list

Comment (inyono) [inline src/assets/source/scripts/modules/serlo_multiple_choice.js]: I don't quite like this spec TBH, should be the other way round. IMHO, positive answers usually don't need individual feedback, but wrong answers do (Why is this answer wrong? What did I do wrong?).

Comment (arekkas) [inline src/assets/source/scripts/modules/serlo_multiple_choice.js]: check line https://github.com/serlo-org/athene2/pull/333/files/92f98f7b15aeaaaa8128aaed8c05d3c2606391ce..0866e2c332d40ed41d971680430c49ec4fa7f2fd#diff-17a8d29c0d3e3d685c25a2a441b5ba29R36

Comment (arekkas) [inline src/assets/source/scripts/modules/serlo_multiple_choice.js]: and https://github.com/serlo-org/athene2/pull/333/files/92f98f7b15aeaaaa8128aaed8c05d3c2606391ce..0866e2c332d40ed41d971680430c49ec4fa7f2fd#diff-3c17fe4d1a48a24f0510aba5f53472cbR24

Comment (inyono) [inline src/assets/source/scripts/modules/serlo_multiple_choice.js]: ah okay, so it is exactly like I want it to be :)

**Comment by inyono** [comment]
still LGTM

File: src/module/Entity/config/types.config.php [hunk 2]
```diff
@@ -135,6 +147,56 @@
                     'license' => []
                 ]
             ],
+            'multiple-choice-wrong-answer' => [
+                'title' => 'id',
+                'description' => 'content',
+                'components' => [
+                    'repository' => [
+                        'form' => __NAMESPACE__ . '\Form\SingleChoiceAnswerForm',
```
Comment (Knorrke) [inline src/module/Entity/config/types.config.php]: shouldn't this be '\Form\MultipleChoiceAnswerForm'?

File: src/module/Entity/config/types.config.php [hunk 3]
```diff
@@ -135,6 +147,56 @@
...
+                    ],
+                    'license' => []
+                ]
+            ],
+            'multiple-choice-right-answer' => [
+                'title' => 'id',
+                'description' => 'content',
+                'components' => [
+                    'repository' => [
+                        'form' => __NAMESPACE__ . '\Form\SingleChoiceAnswerForm',
```
Comment (Knorrke) [inline src/module/Entity/config/types.config.php]: Same here, should probably be '\Form\MultipleChoiceAnswerForm'

File: src/module/Entity/config/types.config.php (hunk 2 above)
Comment (arekkas) [inline src/module/Entity/config/types.config.php]: yup

**Pull request merged**